### ISSUE TITLE
Flutter Web support

### ISSUE THREAD

**Issue by opensourcegps** [vukoye/xmpp_dart#30]
Can this work with Flutter Web? 
Currently it shows error:
```
Socket error: Unsupported operation: InternetAddress
```
Probably because dart:io is not available on the web and it should use HtmlWebSocketChannel. Here is package that explains it a bit better: https://pub.dev/packages/web_socket_channel

Any help is appreciated.

**Comment by vukoye** [comment]
Yes you are right that's because dart:io library is not for the web.

I am not sure how SecureSocket could be implemented with this library as API is bit different, but I encourage you to try, as I have no experience with Flutter Web.
